test/mitmproxy/data/har_files/head-content-length.json

```diff
@@ -0,0 +1,193 @@
+                [
+                    "etag",
+                    "\"a296c6e224c118b0d08cd77e8c08f4b1\""
+                ],
+                [
+                    "x-amz-request-id",
+                    "aeb4d3335548af85"
+                ],
+                [
+                    "x-amz-id-2",
+                    "aN65jxTFgNrlm8zEJMNdk7mYLYwUwTzh0"
+                ],
+                [
+                    "x-amz-version-id",
+                    "4_z179c51e67f11a0ad8f6c0018_f10789ff3151435c8_d20230810_m113900_c005_v0501001_t0045_u01691667540984"
+                ],
+                [
+                    "content-type",
+                    "application/octet-stream"
+                ],
+                [
+                    "cache-control",
+                    "max-age=365000000, immutable, public"
+                ],
+                [
+                    "accept-ranges",
+                    "bytes"
+                ],
+                [
+                    "date",
+                    "Tue, 12 Dec 2023 16:23:16 GMT"
+                ],
+                [
+                    "age",
+                    "2335275"
+                ],
+                [
+                    "x-served-by",
+                    "cache-iad-kcgs7200038-IAD, cache-lck10926-LCK"
+                ],
+                [
+                    "x-cache",
+                    "HIT, HIT"
+                ],
+                [
+                    "x-cache-hits",
+                    "21459, 139679"
+                ],
+                [
+                    "x-timer",
+                    "S1702398196.107663,VS0,VE0"
+                ],
+                [
+                    "strict-transport-security",
+                    "max-age=31536000; includeSubDomains; preload"
+                ],
+                [
+                    "x-frame-options",
+                    "deny"
+                ],
+                [
+                    "x-xss-protection",
+                    "1; mode=block"
+                ],
+                [
+                    "x-content-type-options",
+                    "nosniff"
+                ],
+                [
+                    "x-permitted-cross-domain-policies",
+                    "none"
+                ],
+                [
+                    "x-robots-header",
+                    "noindex"
+                ],
+                [
+                    "x-pypi-file-python-version",
+                    "py3"
+                ],
+                [
+                    "x-pypi-file-version",
+                    "4.66.1"
+                ],
+                [
+                    "x-pypi-file-package-type",
+                    "bdist_wheel"
+                ],
+                [
+                    "x-pypi-file-project",
+                    "tqdm"
+                ],
+                [
+                    "content-length",
+                    "78258"
+                ]
+            ],
+            "contentLength": 0,
```

Comment:
And here is the regression test, showing that we _preserve_ a `content-length` header even when the response has no content.